tools/db_crashtest.py

```diff
@@ -120,6 +120,9 @@
     "optimize_filters_for_memory": lambda: random.randint(0, 1),
     "partition_filters": lambda: random.randint(0, 1),
     "partition_pinning": lambda: random.randint(0, 3),
+    # `get_update_since_one_in` requires no sequence number hole in all WALs
+    # To simplify santization, we enable or disable `get_update_since_one_in` consistently across runs
+    "get_update_since_one_in": random.choice([0, 1000000]),
```

Comment:
Since it is a relatively uncommon/simpler function and testing it prevents us from testing common/complicated functions (disableWAL writes, IngestExternalFile()), I think we should enable it much less frequently. Maybe one in ten or even less frequent.